docs/openapi.yml

```diff
@@ -449,7 +449,7 @@ components:
           type: string
       example:
         name: wework
-        createdAt: 2018-10-04T15:01:23.045Z
+        createdAt: 2019-05-09T00:24:57.609125Z
```

Comment:
Stop working past midnight :)!

Reply:
Actually, I should provided better timestamps here. Also, will try not to :)